### ISSUE TITLE
Question: llama3

### ISSUE THREAD

**Issue by ruizcrp** [epfLLM/meditron#44]
Hi,
On the meta-site https://ai.meta.com/blog/llama-2-3-meditron-yale-medicine-epfl-open-source-llm/
I saw that you also published a model based on llama3? However, I could not find it and the link provided on the meta-site does not work: https://meditron-ddx.github.io/llama3-meditron.github.io/
I was just curious, if you are going to publish it?
Kind regards!

**Comment by lordpba** [comment]
Yes, I am very interested, please let us know with more info about this, thanks!

**Comment by YuniorGlez** [comment]
Same question here, looking for the version with LLama3 :P 

I found that website https://www.meditron.io/  but not info about a model based on llama3 ;( 

EDITED: I found that => https://huggingface.co/ruslanmv/Medical-Llama3-8B
